packages/augur-ui/src/modules/events/actions/log-handlers.ts

```diff
@@ -231,7 +231,9 @@ export const handleOrderCreatedLog = (log: Logs.ParsedOrderEventLog) => (
     log.orderCreator,
     getState().loginAccount.address
   );
-  if (isUserDataUpdate) {
+  // Ignore alerts for pre-liquidity orders here.
+  // They are handled elsewhere so they can be grouped as a single alert.
+  if (isUserDataUpdate && log.tradeGroupId === NULL_ADDRESS) {
```

Comment:
tradeGroupId isn't null for normal orders

Reply:
making to so the normal orders aren't being updated with this info and their toasts don't show up